gravitee-am-ui/src/app/domain/settings/self-service-account/self-service-account.component.html

```diff
@@ -28,6 +28,30 @@ <h1>Self-service account management</h1>
           </mat-slide-toggle>
           <mat-hint style="font-size: 75%;">Enable application end-user to manage its account through self-service account API endpoints.</mat-hint>
         </div>
+
+        <div class="gv-form-section">
+          <div class="gv-form-section-title">
+            <h5>Change password Settings</h5>
+            <mat-divider></mat-divider>
+          </div>
+          <div style="display: flex; flex-direction: column; margin-top: 20px" >
+            <mat-label>Expiration period</mat-label>
+            <mat-form-field appearance="outline" floatLabel="always" style="width: 300px; ">
+                <mat-icon matPrefix style="margin-right: 10px;">timer</mat-icon>
+                <input matInput type="number" placeholder="Token age" min="0" name="tokenAge" [(ngModel)]="tokenAge" (ngModelChange)="setTokenAge()" >
+            </mat-form-field>
+            <mat-hint style="font-size: 75%;; margin-top: -20px;">Set the duration (seconds) a user will have before the token expires.</mat-hint>
+          </div>
+          <div style="display: flex; flex-direction: column; margin-top: 20px;">
+            <mat-slide-toggle
+              (change)="oldPasswordRequired($event)"
+              [checked]="isOldPasswordRequired()">
+              Provide old password
+            </mat-slide-toggle>
+            <mat-hint style="font-size: 75%;">User will be asked to enter their old password when setting a new one</mat-hint>
```

Comment:
Suggestion: "User needs to provide old password while setting the new one"

Reply:
I used wording provided by the Mock screen.